src/pages/index.page.tsx

```diff
@@ -42,7 +42,7 @@ export default function Home(data: {
         <CardLayoutCollection
           cardLayouts={cardLayoutData}
           isOnHomePage={true}
-          filter={(e: CardLayoutData) => e.course.isFeatured === false}
+          filter={(e: CardLayoutData) => e.course.id !== featuredCourse.id}
```

Comment:
Thought that it would be better to have it match the actual ID of the featured course object we have in case no featured course was actually set in the table